app/models/docker_parameter.rb

```diff
@@ -0,0 +1,21 @@
+class DockerParameter < ActiveRecord::Base
+  extend FriendlyId
+  friendly_id :key
+  include Parameterizable::ByIdName
+
+  validates_lengths_from_database
+
+  include Authorizable
+  validates :key, :presence => true, :no_whitespace => true
+
+  scoped_search :on => :key, :complete_value => true
+
+  default_scope -> { order("docker_parameters.key") }
+
+  before_validation :strip_whitespaces
```

Comment:
Because whitespace is allowed in value, it's just to remove trailing and leading whitespace.
This was in the original Parameters, if it's not needed I can remove it but then the values '  22' and '22' will not be considered identical.